packages/gatsby-remark-embed-snippet/README.md

```diff
@@ -1,40 +1,115 @@
 # gatsby-remark-embed-snippet
 
-Embeds the contents of specified files within Prism-formatted HTML blocks.
+Embeds the contents of specified files as code-snippets.
 
-## Overview
+## Installation
+
+**Note**: This plugin depends on [gatsby-remark-prismjs](https://www.gatsbyjs.org/packages/gatsby-remark-prismjs/) and [gatsby-transformer-remark](https://www.gatsbyjs.org/packages/gatsby-transformer-remark/) plugins
+
+```shell
+# npm
+npm install --save gatsby-remark-embed-snippet gatsby-remark-prismjs gatsby-transformer-remark
+
+# yarn
+yarn add gatsby-remark-embed-snippet gatsby-remark-prismjs gatsby-transformer-remark
+```
+
+## Configuration
+
+> **Important**: _You must add `gatsby-remark-embed-snippet` before `gatsby-remark-prismjs` in your plugins array!_
+> Otherwise, this plugin will not work because the code-snippet files first need to be inlined before it can be transformed into code blocks.
```

Comment:
```suggestion
> Otherwise, this plugin will not work because the code snippet files first need to be inlined before they can be transformed into code blocks.
```